concrete/single_pages/dashboard/system/calendar/colors.php

```diff
@@ -26,8 +26,8 @@
 
     <fieldset>
         <legend><?=t('Category Colors')?></legend>
-
-        <?php if (count($topics)) {
+    
+        <?php if (is_array($topics) && !empty($topics)) {
```

Comment:
I'd swap `is_array` and `!empty`, so that we don't have a PHP warning if `$topics` is undefined - see https://3v4l.org/WAcKE vs https://3v4l.org/Rnn2e